domain/schema/model/sql/0011-storage.sql

```diff
@@ -147,7 +147,8 @@ CREATE TABLE storage_unit_owner (
 );
 
 CREATE TABLE storage_attachment (
-    storage_instance_uuid TEXT NOT NULL PRIMARY KEY,
+    uuid TEXT NOT NULL PRIMARY KEY,
+    storage_instance_uuid TEXT NOT NULL,
```

Comment:
```suggestion
    storage_instance_uuid TEXT NOT NULL UNIQUE,
```

Reply:
Since we frequently query this table by storage_instance_uuid and unit_uuid, I added indexes on those columns instead.